### PR TITLE
WP-94-selenium-test-movie-search

### PR DESCRIPTION
**Pull request by shahan27**
Add the selenium test for search results, let me know if any changes should be made. Thanks :)
Also attached a readme so it won't be a pain to run hopefully.

### CODE REVIEW

**Comment by PredatorFeesh** [comment]
I'm thinking, and I know this sounds ugly, but a 5th middleman repo (eh the cons of not having EC2s right :cry:) We can treat that 5th middleman repo as our test repo, and make it run routine tests. Would be interesting to see if this works :laughing: after we make this more config dependent ofc (won't be too had a fix - seems more of a gaction thing. you can keep devving more tests.)

Let me know

**Comment by PredatorFeesh** [comment]
Or maybe we should run this with every deploy instead of on a timer - in that case, we can make `WatchParty/frontend/selenium` folder and put the tests in there. The rest (config and stuff) is similar to last comment.

**Comment by PredatorFeesh** [comment]
*sidenote, when we decide, don't close this branch and reopen. Instead, do `git mv` and then your commit and push to move all the files. `git mv` is like regular command line `mv`, but it also lets git know a move was done

**Comment by shahan27** [comment]
Just moved the files around. :)

**Review by watchpartyappmain — APPROVED** [review]
Let's get this in asap :D

File: WatchParty/client/selenium/movieResultTest.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from selenium import webdriver
+import unittest
+import time
+
+class ResolutionTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Chrome(executable_path='/Users/Shahan/Desktop/chromedriver')
```
Comment (watchpartyappmain) [inline WatchParty/client/selenium/movieResultTest.py:9]: I just noticed, this is a big oof! We need paths to be relative and not absolute! Also, the server will not be able to find this chromedriver! Get it into this branch as part of this commit, and instead change it to 
`executable_path='./chromedriver' `Also it's @PredatorFeesh

**Comment by PredatorFeesh** [comment]
I messed up by merging this in already and then reverting it 😞 open this branch again for merge (not this PR)

**Pull request merged**